src/edu/csus/ecs/pc2/ui/AutoJudgeStatusFrame.java

```diff
@@ -138,10 +143,14 @@ private JPanel getMessagePanel() {
     private JButton getStopAutoJudgingButton() {
         if (stopAutoJudgingButton == null) {
             stopAutoJudgingButton = new JButton();
-            stopAutoJudgingButton.setText("Stop AutoJudging");
+            stopAutoJudgingButton.setText("Stop Auto Judging");
```

Comment:
I'm not sure about this change (although I grant it's trivial).  We refer to the components doing automatic judging as "AJ"s, not "A J"s, and the relevant classes are named like `AutoJudgeStatusFrame`, not `Auto_Judge_Status_Frame`.

I guess it's a personal style thing...  but I'll bet there are _dozens_ of places where "AutoJudging" is used, not "Auto Judging"...

Reply:
And I'm not sure why I even made that change - it was June of last year :-) I probably thought since it was a GUI thing it was more readable to a user to be "Auto Judging" instead of "AutoJudging" - "AutoJudgxxxx" is internal.  That's all I can think of.  I can put it back.

Reply:
Fixed in next push.